src/include/btmem.h

```diff
@@ -127,6 +127,7 @@ __wt_page_header_byteswap(WT_PAGE_HEADER *dsk)
  */
 struct __wt_addr {
     /* Validity window */
+    wt_timestamp_t oldest_durable_ts;
     wt_timestamp_t newest_durable_ts;
```

Comment:
I prefer "newest_durable_ts" to be after "oldest_start_txn", that way all the oldest will be first and then the newest. I think that visualisation will be better during debugging as we can think first three are oldest and then newest.

Reply:
Fixed.